project-templates/feature_request.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Feature request
+about: Suggest new feature
+title: "feat: "
+labels: ''
+assignees: ''
+
+---
+
+### Title
+
+*Example: Create Members GET API*
```

Comment:
Do we need to duplicate the title in the description? The title will be automatically populated with the `"feat: "` as you put on 4th line of the doc.

Reply:
Yes, less is more - I will remove this line.